views/list-capabilities.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Capabilities</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+</head>
+<body>
+<header class="bg-dark text-light">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <h1>Capabilities</h1>
+            </div>
+        </div>
+    </div>
+</header>
+<main class="container mt-4">
+    <section id="job-roles" class="mt-4">
+        <h2>Capabilities</h2>
+        {% if capabilities and capabilities.length > 0 %}
+        <table class="table">
+            <thead>
+            <tr>
+                <th scope="col">Capabilities</th>
+                <th scope="col">More Info</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for capability in capabilities %}
+            <tr>
+                <td> {{capability.name}}</td>
+                <td><a class="btn btn-primary" href="{{role.link}}" style="background-color: #0072ed" role="button">Link</a></td>
```

Comment:
Where does role come from here?

Reply:
Nice spot @Patrick-Mc-Grath, this should be `capability.link`

```suggestion
                <td><a class="btn btn-primary" href="{{capability.link}}" style="background-color: #0072ed" role="button">Link</a></td>
```